### PR TITLE
SYS-3640 update LogLifted event signature

### PR DESCRIPTION
**Pull request by MBrozhko34**
## Proposed changes

Updated LiftedData parameters and LogLifted event signature removing tier 1 address.

## Type of change/Merge

🚨What type of change is this PR? </br>
_Put an `x` in the boxes that apply_

- [ ] Release 
  - [ ] Increase versions 
  - [x] Baseline tests passed  
  - Release type:
    - [ ] Major release 
    - [ ] Minor release 
    - [ ] Patch release 

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [x] You describe the purpose of the PR, e.g.:
  - What does it do?
  - Highlight what important points reviewers should know about;
  - Indicates if there is something left for follow-up PRs.
- [ ] Documentation updated
- [ ] Business logic tested successfully
- [ ] Verify First, Write Last: In Substrate development, it is important that you always ensure preconditions are met and return errors at the beginning. After these checks have completed, then you may begin the function's computation.

### CODE REVIEW

**Review by nahuseyoum — APPROVED** [review]

**Review by alexpArtos — COMMENTED** [review]
Without having run the tests, I think there is at least one thing missing in this PR, and that is to update the `signature` for LogLifted around lines 85-87. This code here: 

```
            // hex string of Keccak-256 for LogLifted(address,address,bytes32,uint256)
            ValidEvents::Lifted =>
                H256(hex!("8964776336bc2fa8ecaaf70b6f8e8450807efb1ff78f8b87980707aa821f0ec0")),
```

But this triggered other thoughts:
- the current changes make any previous LogLifted event invalid, even though they still exist on the chain and were valid at that time. Wouldn't it be better to keep both versions and be able to validate the current events and the late events?
- also, shouldn't we run tests with an actual even following the new format, before submitting the PR? Without the signature change, I'm not sure we will be able to validate a new event and would like to have some confirmation of that before accepting the PR

File: primitives/avn-common/src/event_types.rs [hunk 1]
```diff
@@ -208,21 +208,17 @@ impl AddedValidatorData {
...
-    pub sender_address: H160,
     pub receiver_address: H256,
     pub amount: u128,
     pub nonce: U256,
 }
 
 impl LiftedData {
     const TOPIC_CURRENCY_CONTRACT: usize = 1;
-    const TOPIC_INDEX_T1_ADDRESS: usize = 2;
     const TOPIC_INDEX_T2_ADDRESS: usize = 3;
```
Comment (alexpArtos) [inline primitives/avn-common/src/event_types.rs:218]: I think this needs to change as well, now that I go into the details. Now that we don't have the sender data in the event, we lose one of the topics, so what comes after it (the T2 address) moves up one position to be number 2.

Comment (alexpArtos) [inline primitives/avn-common/src/event_types.rs:218]:
```suggestion
    const TOPIC_INDEX_T2_ADDRESS: usize = 2;
```

**Review by alexpArtos — APPROVED** [review]
Conditional approval. See comments

**Comment by MBrozhko34** [comment]
Working on resolving tests

**Review by alexpArtos — APPROVED** [review]
Well done, all the tests pass, I think we're good to go now!

**Pull request merged**